pkg/skaffold/initializer/testdata/init/helm-deployment/README.md

```diff
@@ -0,0 +1,37 @@
+### Example: deploy multiple releases with Helm
+
+You can deploy multiple releases with skaffold, each will need a chartPath, a values file, and namespace.
+Skaffold can inject intermediate build tags in the the values map in the skaffold.yaml.
+
+Let's walk through the skaffold yaml:
+
+We'll be building an image called `skaffold-helm`, and it's a dockerfile, so we'll add it to the artifacts.
```

Comment:
Keep the tenses the same as later.
```suggestion
We are building an image called `skaffold-helm`, and it builds with Docker, so we add it to the artifacts.
```

Reply:
actually we shouldn't be updating this - this was copied directly from https://github.com/GoogleContainerTools/skaffold/blob/master/examples/helm-deployment/README.md :) we can update that later but i'll leave this as is in this PR